### DISCUSSION TITLE
2023/08/28 Task

### DISCUSSION THREAD

**Discussion by hyunnnn98** [2P3S/scrum-dice#95] [General]
# 승현
## 주말에 같이할 것
- [x] #105
- [x] #104 
- [x] #99
- [x] #101
- [x] #102
- [ ] #90

## 평소에 해두면 좋은 것
- [ ] 그 외 사소한 부분
    - [ ] Error handling
        - 실제 통합 테스트 해보면서 동작에 크게 지장없으면 일단 Issue 로 만들고 Pass
        - 그 외에 에러는 실시간으로 확인
        - 에러 메시지 다국어로 할 거 정리 필요 (백에서 받은 메시지를 각 언어로 변환)
  - [ ] #108 
  - [ ] #109
  - [ ] 다크 모드 글자 폰트 수정
    -  우선도가 낮기 때문에 off 로 설정해두기

# 재순
## 주말에 같이할 것
- [x] AWS 서버 최종결정 -> 프리티어, 프론트엔드는 vercel로 배포
	![image](https://github.com/2P3S/2p3s-scrum-dice/assets/53788601/bb8387db-5909-463d-b30b-32ce1ba832c1)
- [ ] Error Handling 설계 및 구현
	- [x] join-request
	- [x] create-vote
	- [x] update-vote-name
	- [ ] submit-card
	- [ ] open-card
- [ ] Vercel 프론트 배포해보기!

# 승형
## 주말에 같이할 것
- [x] #54 
- [ ] 한국어, 영어 페이지 별로 변수 작성
  - [ ] 영어로 먼저 작성해두기
  - [x] common.js 를 페이지 단위로 파일 나누기
  - [ ] 일본어는 승현이가
  - [x] 변수 이름은 그냥 한글로 설정

## 평소에 해두면 좋은 것
- [ ] 리팩토링 필요한 곳 찾기
- [ ] 서버에서 조금 더 수정할만한 설정이 있는지 찾아보기
- 그 외에 Issue 에서 할만한 것 있는지 보기

**Comment by JeongJaeSoon** [comment]
Vercel 배포 순서 참고 자료
https://2dowon.github.io/docs/react/deploy-by-vercel/

**Comment by JeongJaeSoon** [comment]
Vercel 계정 잠겨있었기 때문에 문의 넣음


> registration@vercel.com
>
> Dear customer support,
> 
> My name is 2P3S, an IT engineer in Korea.
> 
> I tried signing up to vercel by using my GitHub account for learning.
> However it's blocked with the following message.
> "Error:This user account is blocked"
> 
> So please let me know
> ・Why it's blocked?
> ・How can I succeed in signing up?
> 
> I look forward to hearing from you.
> 
> Sincerely,
